### PR TITLE
feat/verification status distinction

### PR DESCRIPTION
**Pull request by bennchong**
Validation Bar would show the different distinct status of the workpass

- [x] Expired 
- [x] Expired with legal stay
- [x] Tampered
- [x] Revoked 
- [x] Invalid

### CODE REVIEW

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=37

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=37

File: src/components/profile/ProfileContainer.tsx [hunk 1]
```diff
@@ -57,16 +62,14 @@ export const ProfileContainer = ({
...
-      verifyWorkpassBoolean(workpass).then(isValid => {
-        setValidityStatus(isValid ? statusEnum.VALID : statusEnum.INVALID);
-        if (isValid && !isPreview) {
-          storeTime();
-        } else if (isValid && isPreview) {
-          setPreviewTime(getCurrentDateAndTime());
-        }
-      });
-    }
+    verifyWorkpass(workpass).then(status => {
```
Comment (samuellye98) [inline src/components/profile/ProfileContainer.tsx]: how about if(workpass) to check if there is an existing workpass

Comment (bennchong) [reply src/components/profile/ProfileContainer.tsx]: how about no

Comment (bennchong) [reply src/components/profile/ProfileContainer.tsx]: jk yes

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=37

File: src/components/profile/ProfileContainer.tsx (hunk 1 above)
Comment (bennchong) [reply src/components/profile/ProfileContainer.tsx]: @samuellye98  done :)

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=37

**Comment by yehjxraymond** [comment]
Expo QR
Published to https://exp.host/@workpasssg/sg-workpass?release-channel=37

**Pull request merged**